src/repos/MagazineRepo.ts

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-underscore-dangle */
+import Filter from 'bad-words';
+import { ObjectId } from 'mongodb';
+import { Magazine, MagazineModel } from '../entities/Magazine';
+import {
+  DEFAULT_LIMIT,
+  DEFAULT_OFFSET,
+  FILTERED_WORDS,
+  MAX_NUM_DAYS_OF_FEATURED_MAGAZINES,
+  IS_FILTER_ACTIVE,
+} from '../common/constants';
+
+function isMagazineFiltered(magazine: Magazine) {
+  if (IS_FILTER_ACTIVE) {
```

Comment:
when would we disable this filter? shouldn't this be an environment variable so that we can flip it on and off without having to make a new push?

Reply:
We want to disable it now because there isn't any covid content policy on the apple store anymore

Reply:
It was disabled in another pr, but I can change it to an env variable there